web_src/js/components/ViewFileTreeItem.vue

```diff
@@ -58,7 +58,8 @@ const doGotoSubModule = () => {
   >
     <!-- submodule -->
     <div class="item-content">
-      <SvgIcon class="text primary" name="octicon-file-submodule"/>
+      <!-- eslint-disable-next-line vue/no-v-html -->
+      <span class="tw-contents" v-html="item.entryIcon"/>
```

Comment:
Why not avoid `v-html` by passing dynamic `name` into `SvgIcon`?

Reply:
As far as I know, svg-icon are statically defined, and currently, file icons are uniformly rendered dynamically from the backend to support dynamic updates of the theme library!

Reply:
So `item.entryIcon` is the actual SVG source code?

Reply:
Yeah, it's fine. The only thing I don't like about this is the `tw-contents` and the additional `<span>` wrapper this introduces. I think we could get Vue to render raw HTML without a wrapper with solutions like these:

https://stackoverflow.com/a/67771171/808699
https://stackoverflow.com/a/69354385/808699